packages/p2p/src/components/app.jsx

```diff
@@ -44,10 +44,11 @@ class App extends Component {
                 <main className='deriv-p2p'>
                     <nav>
                         <Tabs active_index={active_index}>
-                            <div label={localize('Buy/sell')}>
+                            <div label={localize('Buy / Sell')}>
                                 <BuySell />
                             </div>
-                            <div label={localize('Orders')}>
+                            {/* TODO: Change this count prop after BE API is ready */}
```

Comment:
can you use the tag `[p2p-replace-with-api]` for these todos? also please avoid the wording `BE`